### PR TITLE
fixing for mac and linux as well

### PR DESCRIPTION
**Pull request by akhiljns**

### CODE REVIEW

File: hypervisor/virtualbox/virtualbox.go [hunk 1]
```diff
@@ -51,8 +50,11 @@ func (*virtualbox) Create(imageName, vmName string, cpus int, mem int) error {
 		"--memory", strconv.Itoa(mem),
 	}
 
+	// Wrap each argument in quotes if it contains spaces
 	for i, arg := range importArgs {
-		importArgs[i] = `"` + arg + `"`
+		if strings.Contains(arg, " ") {
+			importArgs[i] = `"` + arg + `"`
+		}
```
Comment (mhewedy) [inline hypervisor/virtualbox/virtualbox.go:57]: do we need to revert the old code (related to CPU and memory flags)?

image

Comment (mhewedy) [reply hypervisor/virtualbox/virtualbox.go:57]: I suggest to revert the old code, and then just fix the parts that contains the filesystem path

Comment (akhiljns) [reply hypervisor/virtualbox/virtualbox.go:57]: I'm getting error 

_VBoxManage: error: Invalid command '"import"'_

I guess mac is not able to take the import argument in string format like windows

cpus and memory flags are not the issue

Comment (akhiljns) [reply hypervisor/virtualbox/virtualbox.go]: @mhewedy reverted to old style of code just updating the paths to include quotes


now tested on windows, mac and linux

Comment (mhewedy) [reply hypervisor/virtualbox/virtualbox.go]: Thanks

**Pull request merged**